simulert/cli.py

```diff
@@ -0,0 +1,73 @@
+import os
+import runpy
+
+import click
+from simulert import getAlerter
+from simulert.handlers import Emailer, Slacker
+
+alerter = getAlerter()
+
+
+@click.group()
+@click.option("-e", "--email", help="attach email handler", is_flag=True)
+@click.option("-s", "--slack", help="attach slack handler", is_flag=True)
+@click.option(
+    "--simulertSlackToken",
+    help="the token for the slack-bot used to send messages from.",
+)
+@click.option(
+    "--simulertSlackUsername",
+    help="the username of the slack user to send messages to.",
+)
+@click.option(
+    "--simulertEmailHost", help="the host address of the email server to send from."
+)
+@click.option(
+    "--simulertEmailPort", help="the connection port of the email server to send from."
+)
+@click.option(
+    "--simulertEmailAuthentication",
+    help="comma-separated username and password to authenticate to the email server.",
+)
+@click.option(
+    "--simulertEmailSender", help="comma-separated sender name and email address"
+)
+@click.option(
+    "--simulertEmailRecipient", help="comma-separated receiver name and email address"
+)
+def cli(
+    email,
+    slack,
+    simulertslacktoken=None,
+    simulertslackusername=None,
+    simulertemailhost=None,
+    simulertemailport=None,
+    simulertemailauthentication=None,
+    simulertemailsender=None,
+    simulertemailrecipient=None,
+):
+    if not slack and not email:
+        print(
+            "Try 'simulert --help' for help.\n\nError: Please specify at least one handler."
+        )
+        exit(0)
+    if slack:
+        slacker = Slacker(simulertslacktoken, simulertslackusername)
+        alerter.add_handler(slacker)
+    if email:
+        emailer = Emailer(
+            simulertemailhost,
+            simulertemailport,
+            simulertemailauthentication,
+            simulertemailsender,
+            simulertemailrecipient,
+        )
+        alerter.add_handler(emailer)
+
+
+@cli.command()
+@click.option("-n", "--name", help="simulation name")
+@click.argument("method", required=True)
+def run(name, method):
```

Comment:
what do you think about having this run from main so ` simulert -s  ~/hello_world.py` would work?
I'm not sure independent commands add anything when there's only one.

Reply:
I wasn't sure if we wanted scope for adding more commands further down the line such as `test_connection`? If not then I can certainly refactor the `run` command into main.

Reply:
I see your point and it's a fair consideration. How about having `run` as the default command?
https://click.palletsprojects.com/en/7.x/commands/#group-invocation-without-command

Reply:
I think that's a good comprise, I'll update and push the changes

Reply:
Update: this is actually not so easy since the `run` sub-command requires arguments to be passed. Since there is no separator distinguishing the root arguments from the sub-command arguments they all get treated as root arguments and so there is no way to specify arguments for `run` when you call it by default as opposed to specifying it directly. The `run` command requires a python file path to run.

Reply:
Okay, no worries. Let's leave it like this for now then.